mobile_bkend/flamingo-proot-distro.sh

```diff
@@ -0,0 +1,20 @@
+# Added By Sanjay
+DISTRO_NAME="Ubuntu (focal) for Flamingo"
```

Comment:
`Flamingo` -> `Swan`?

Reply:
I think Flamingo is fine; not sure about anonymity requirements